custom_components/vikunja/__init__.py

```diff
@@ -36,7 +35,8 @@ async def async_setup_entry(hass, entry):
         return False
 
     # Initialize Vikunja API client
-    vikunja_api = VikunjaAPI(base_url, token, strict_ssl)
+    client = get_async_client(verify_ssl=strict_ssl)
```

Comment:
Getting an error from this line when running. Did you verify in a dev environment yourself?

I'm seeing
```
Traceback (most recent call last):
2025-09-10 16:24:56   File "/usr/src/homeassistant/homeassistant/config_entries.py", line 637, in __async_setup_with_context
2025-09-10 16:24:56     result = await component.async_setup_entry(hass, self)
2025-09-10 16:24:56              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
2025-09-10 16:24:56   File "/config/custom_components/vikunja/__init__.py", line 38, in async_setup_entry
2025-09-10 16:24:56     client = get_async_client(verify_ssl=strict_ssl)
2025-09-10 16:24:56 TypeError: get_async_client() missing 1 required positional argument: 'hass'
```

Seems just adding this resolves it
```suggestion
    client = get_async_client(hass, verify_ssl=strict_ssl)
```